requirements.txt

```diff
@@ -1,4 +1,5 @@
 boto3==1.20.26
+botocore==1.23.54
```

Comment:
Should these two be the same version?

Reply:
They should be fine as different versions, I looked online and saw the [latest version of boto3](https://github.com/boto/boto3/tags) 1.25.3 and [the latest version of botocore](https://github.com/boto/botocore/tags) is 1.28.3. I imagine if they had to be the same version the latest versions would be the same, and weren't any conflicts on my machine when I ran nagbot (I have those two versions running).